pkgs/development/compilers/tinygo/default.nix

```diff
@@ -161,5 +161,8 @@ buildGoModule rec {
     description = "Go compiler for small places";
     license = licenses.bsd3;
     maintainers = with maintainers; [ Madouura muscaln ];
+    # vendor can't be recreated and fails with:
+    #   error: illegal path references in fixed-output derivation '/nix/store/......-tinygo-0.26.0-go-modules.drv'
```

Comment:
@SuperSandro2000 can you explain how you got this error ? 

I recently tried updating tinygo and I kept getting vendor hash mismatch, so maybe its related

Reply:
just run the following ``nix-build -A tinygo.go-modules --check``

Reply:
@SuperSandro2000 thanks for your hint, helped me to track down the problem https://github.com/NixOS/nixpkgs/pull/245445